lib/mutant/reporter/cli/printer.rb

```diff
@@ -3,10 +3,34 @@ class Reporter
     class CLI
       # CLI runner status printer base class
       class Printer
-        include AbstractType, Delegator, Adamantium::Flat, Concord.new(:output, :object), Procto.call(:run)
+        include AbstractType,
+                Adamantium::Flat,
+                Concord.new(:output, :object),
+                Procto.call(:run)
 
         private_class_method :new
 
+        # Create delegators to object
+        #
+        # @return [undefined]
+        def self.delegate(*names)
```

Comment:
> Heh, I realize now you removed Mutant::Delegator for this PR. Why did you do that?
> 
> I think it's fine to separate plumbing code from domain knowledge.

Short reason: It was easier to mutation cover at the call side than mutation covering the extracted module.